schemas/generic-serialization.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://lionweb.org/generic-serialization.schema.json",
+  "title": "Generic serialization format",
+  "type": "array",
+  "items": {
+    "$ref": "#/$defs/SerializedNode"
+  },
+  "$defs": {
+    "Id": {
+      "type": "string",
+      "minLength": 1
+    },
+    "SerializedNode": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string"
+        },
+        "id": {
+          "$ref": "#/$defs/Id"
+        },
+        "properties": {
+          "type": "object"
+        },
+        "children": {
+          "type": "object"
+        },
+        "references": {
```

Comment:
I don't know JsonSchema good enough, but does this take the discussion about NodeInfo into account?
https://github.com/LIonWeb-org/organization/issues/36

Reply:
Do you mean the possibility of providing (additional) info that {c|sh}ould help with resolution? I didn't here (yet): either a reference resolves through ID, or it doesn't and it's serialized as `null`. We can always add a third option for `resolveInfo` later on as soon as we've formulated a sensible format for that, right?

Reply:
from the issue:
> How to encode
>For references, we provide an object like targetId: resolveInfo.
If no resolve info is available, it's set to null.
>The value can be any string.

So I'd think `references` is a map (or whatever that's called in json) of
```
targetid: id(i.e. string) -> resolveinfo: string | null
```

Reply:
OK, I think I don't understand the info in issue #36, then. How can we clarify that?

I have to work on (de-)serialization anyway, so more is coming, and we can still make course changes.

Reply:
I made issue #16 for this. I'd rather be able to merge this now to `main`, and pick up issue #16 in a separate branch.